modules/apps/site/site-cms-site-initializer/test/js/main/dashboard/ContentCard.test.tsx

```diff
@@ -0,0 +1,54 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import '@testing-library/jest-dom/extend-expect';
+import {render, screen} from '@testing-library/react';
+import React from 'react';
+
+import {ContentCard} from '../../../../src/main/resources/META-INF/resources/js/main/dashboard/components/ContentCard';
+
+describe('[CMS Dashboard] Components: ContentCard', () => {
+	it('renders correctly', () => {
+		render(<ContentCard />);
+
+		const Title = screen.getByText('CONTENT');
+		expect(Title).toBeInTheDocument();
+
+		const Description = screen.getByText(
+			'total-number-of-content-assets-created-in-your-spaces'
```

Comment:
We must wait for the final language key value approved by Brian before sending this PR forward.